merlin/celery.py

```diff
@@ -82,7 +82,7 @@
 
 # set task priority defaults to prioritize workflow tasks over task-expansion tasks
 task_priority_defaults = {
-    "task_queue_max_priority": get_priority(Priority.high),
+    "task_queue_max_priority": 10,
```

Comment:
are we sure about this?

Reply:
I haven't tested this with a redis broker, but my gut feeling is that when asking for "max priority", it wants the highest integer value, to know how many queues to make.